package.json

```diff
@@ -20,6 +20,10 @@
     "node": "4.4.x",
     "npm": "3.9.x"
   },
+  "browserslist": [
```

Comment:
It looks like `autoprefixer` will actually use this exact config! https://github.com/postcss/autoprefixer#options